web-common/src/features/dashboards/pivot/PivotTable.svelte

```diff
@@ -12,9 +12,17 @@
   import type { Readable } from "svelte/motion";
   import { derived } from "svelte/store";
   import type { PivotDataRow, PivotDataStore } from "./types";
+  import {
+    createVirtualizer,
+    defaultRangeExtractor,
+  } from "@tanstack/svelte-virtual";
 
   export let pivotDataStore: PivotDataStore;
 
+  const OVERSCAN = 80;
+  const ROW_HEIGHT = 24;
```

Comment:
24 seems a bit too tight for row height. The mock has height as 28 for each row?

Reply:
Where are you seeing 28? The cells in Figma are 24.

Reply:
Oh, you are right!